css/at-rules/keyframes.json

```diff
@@ -167,8 +167,8 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": false,
-                "impl_url": "https://webkit.org/b/222295"
+                "version_added": "26",
+                "impl_url": "https://webkit.org/b/286920"
```

Comment:
Nit: Once resolved, we usually no longer track `impl_url`.

But this could probably be auto-fixed by our linter, so let's keep this in, especially as the `impl_url` changed.